src/components/navigation-header/components/product-page-content/index.tsx

```diff
@@ -141,34 +142,17 @@ const ProductPageHeaderContent = () => {
 				</Link>
 			</div>
 			{isBetaProduct && (
-				<div className="g-hide-on-mobile g-hide-on-tablet">
+				<div className={sharedNavStyles.leftSideDesktopOnlyContent}>
 					<nav className={sharedNavStyles.nav}>
 						<ul className={sharedNavStyles.navList}>
-							{[
-								{ label: 'Home', pathSuffix: '' },
-								{
-									label: 'Documentation',
-									iconColorTheme: currentProduct.slug,
-									items: currentProduct.rootDocsPaths.map(
-										(rootDocsPath: RootDocsPath) => {
-											return {
-												icon: rootDocsPath.iconName,
-												label: rootDocsPath.name,
-												path: `/${currentProduct.slug}/${rootDocsPath.path}`,
-											}
-										}
-									),
-								},
-								{ label: 'Tutorials', pathSuffix: 'tutorials' },
-								{ label: 'Install', pathSuffix: 'downloads' },
-							].map((navItem) => {
-								const { items, label } = navItem
+							{getNavItems(currentProduct).map((navItem) => {
+								const { type, label } = navItem
 								const ariaLabel = `${currentProduct.name} ${label}`
 
 								let ItemContent
-								if (items) {
+								if (type == 'submenu') {
 									ItemContent = PrimaryNavSubmenu
-								} else {
+								} else if (type == 'link') {
 									ItemContent = PrimaryNavLink
 								}
```

Comment:
The `type` abstraction doesn't feel super necessary here.

Reply:
Sounds good - have removed 👍 I found I couldn't use `items` directly without a type warning, so have used `navItem.hasOwnProperty('items')`.